eo-parser/src/main/java/org/eolang/parser/XeEoListener.java

```diff
@@ -1110,6 +1097,22 @@ public void exitReversed(final EoParser.ReversedContext ctx) {
         this.objects.enter().xprop("base", "concat('.',@base)").leave();
     }
 
+    @Override
+    public void enterAname(final EoParser.AnameContext ctx) {
+        this.objects
+            .enter()
+            .prop(
+                "name",
+                String.format("OBJ-%s", UUID.randomUUID())
```

Comment:
@maxonfjvipon a much safer way would be to use the line number in the file - it is definitely unique.